activelearning/dataset/ocp.py

```diff
@@ -132,7 +161,8 @@ def __init__(
             )
 
     def maxY(self):
-        return 10  # -> TODO: what are the actual bounds?
+        # return 10  # -> TODO: what are the actual bounds?
+        return 5  # when target is standard normalized
```

Comment:
How come the min and max are the same?

Reply:
it's normalized

Reply:
I still don't get it, but ok I guess it doesn't matter.